i18n/enu/out/debug.i18n.json

```diff
@@ -1,10 +1,10 @@
 {
-	"autolispext.debug.nodap": "doesn’t exist. Verify that the file exists in the same folder as that for the product specified in the launch.json file.", 
-	"autolispext.debug.noacad": "doesn’t exist. Verify and correct the folder path to the product executable.", 
-	"autolispext.debug.acad.nosupport": "This instance of AutoCAD doesn’t support debugging AutoLISP files, use a release later than AutoCAD 2020.",
-	"autolispext.debug.launchjson.path": "Specify the absolute path to the product with the Path attribute of the launch.json file.",
-	"autolispext.debug.prod.path.win":"Specify the absolute path for the product. For example, C://Program Files//Autodesk//AutoCAD//acad.exe.",
-	"autolispext.debug.prod.path.osx": "Specify the absolute path for the product. For example, /Applications/Autodesk/AutoCAD.app/Contents/MacOS/AutoCAD.",
-	"autolispext.debug.prod.path.other":"Specify the absolute path for the product.",
-	"autolispext.debug.noprocess.eror":"No process for which to attach could be found."
+  "autolispext.debug.nodap": "doesn’t exist. Verify that the file exists in the same folder as that for the product specified in the launch.json file.",
```

Comment:
it looks like it removes the json format, so it has so much file changes.

Reply:
I would say it fixes the JSON format because it is all ran through a single standardized JSON formatter; Newtonsoft. The data was essentially read and then written back with just as it was (value wise) but with some structural normalization. 

You'll notice some files had this normalization performed and other didn't. This is because our files had inconsistencies. Now they don't.

Reply:
ok, and it (newtonsoft) get the same output as this extensions? https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
I found [this issue](https://github.com/prettier/prettier/issues/8815) talking about how Prettier defaults to using UTF-16 and that is what I believe Newtonsoft defaults to using. So, encoding should be a non-issue. From the structural aspect, I am fairly certain that is extremely configurable (even the encoding) within Prettier and thus highly subjective. So... I guess that's a yes...

In general the JSON files in question (whole document) are exceptionally primitive (`Dictionary<string, string>`) and no "release" worthy version of a JSON reader should ever misinterpret these. The only real risk should be the encoding or human failures in maintaining the structures that corrupt the read process. I did not test for the human error side, but this app would actually help prevent that. I don't think the encoding could have been screwed up or else I would think Git would have flagged the entirety of every document instead of the structural spacing parts.

Reply:
Thanks! I only don't want to add/remove the blank spaces in different PRs because of the different json formatters.